azure-pipelines.yml

```diff
@@ -92,8 +92,8 @@ jobs:
     parameters:
       name: Ubuntu_Clang9_V8
```

Comment:
Don't forget to change the job name

Reply:
CI changes were not intended for this PR, just reverted them.